### ISSUE TITLE
Update RefSeq?

### ISSUE THREAD

**Issue by asaldivar93** [marbl/Mash#161]
Hi,
I'm using mash to detect contamination in de-novo genome assemblies, together with other tools that work on the latest release.
Is it possible to build a sketch file for the genomes in the latest release of the RefSeq database using a PC with 16Gb RAM? 

If it is, could you share the workflow necessary to do it?
If it is not, is someone willing to do the work and share the file?

Any help will be greatly appreciated

**Comment by Caiyulu-818** [comment]
yes.is the [refseq.genomes.k21.s1000.msh](https://gembox.cbcb.umd.edu/mash/refseq.genomes.k21s1000.msh) is the latest version ?

**Comment by kbessonov1984** [comment]
No, it is quite old. I would advise to create a new sketch. NCBI RefSeq now has 136,409 genome reference assemblies while the sketch has 91282. Sometimes I hit deprecated accession numbers that are removed from new metadata  [assembly_file_manifest.txt](http://ftp.ncbi.nlm.nih.gov/genomes/ASSEMBLY_REPORTS/assembly_summary_refseq.txt)
